README.md

```diff
@@ -3762,3 +4352,412 @@ return Http::withOptions([
 
 Tip given by [@raditzfarhan](https://github.com/raditzfarhan)
 
+### Test that doesn't assert anything 
+Test that doesn't assert anything, just launch something which may or may not throw an exception
+
+```php
+class MigrationsTest extends TestCase
+{
+    public function test_successful_foreign_key_in_migrations()
+    {
+        // We just test if the migrations succeeds or thwors an exception
+        $this->expectNotToPerformAssertions();
+    }
+}
+```
+
+### "Str::mask()" method
+Laravel 8.69 released with "Str::mask()" method which masks a portion of string with a repeated character
+
+```php
+class PasswordResetLinkController extends Controller
+{
+    public function sendResetLinkResponse(Request $request)
+    {
+        $userEmail = User::where('email', $request->email)->value('email'); // username@domain.com
+        
+        $maskedEmail = Str::mask($userEmail, '*', 4); // user***************
+        
+        // If needed, you provide a negative number as the third argument to the mask method,
+        // which will instruct the method to begin masking at the given distance from the end of the string
+        
+        $maskedEmail = Str::mask($userEmail, '*', -16, 6); // use******domain.com
+    }
+}
+```
+
+Tip given by [@Teacoders](https://twitter.com/Teacoders/status/1457029765634744322)
+
+### Extending Laravel classes
+There is a method called macro on a lot of built-in Laravel classes. For example Collection, Str, Arr, Request, Cache, File, and so on.<br>
+You can define your own methods on these classes like this:
+
+```php
+Str::macro('lowerSnake', function (string $str) {
+    return Str::lower(Str::snake($str));
+});
+
+// Will return: "my-string"
+Str::lowerSnake('MyString');
+```
+
+Tip given by [@mmartin_joo](https://twitter.com/mmartin_joo/status/1457635252466298885)
+
+### Can feature
+If you are running Laravel `v8.70`, you can chain `can()` method directly instead of `middleware('can:..')`
+
+```php
+// instead of
+Route::get('users/{user}/edit', function (User $user) {
+    ...
+})->middleware('can:edit,user');
+
+// you can do this
+Route::get('users/{user}/edit', function (User $user) {
+    ...
+})->can('edit' 'user');
+
+// PS: you must write UserPolicy to be able to do this in both cases
+```
+
+Tip given by [@sky_0xs](https://twitter.com/sky_0xs/status/1458179766192853001)
+
+### Temporary download URLs
+You can use temporary download URLs for your cloud storage resources to prevent unwanted access. For example, when a user wants to download a file, we redirect to an s3 resource but have the URL expire in 5 seconds.
+
+```php
+public function download(File $file)
+{
+    // Initiate file download by redirecting to a temporary s3 URL that expires in 5 seconds
+    return redirect()->to(
+        Storage::disk('s3')->temporaryUrl($file->name, now()->addSeconds(5))
+    );
+}
+```
+
+Tip given by [@Philo01](https://twitter.com/Philo01/status/1458791323889197064)
+
+### Display the release version
```

Comment:
@krekas I don't think this tip is worth adding, it's kind of a "common sense" thing